go/stats/opentsdb/backend.go

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Ended up refactoring this package a bit, mostly moving stuff to their own files. Will call out which stuff was moved vs. new.